netsim/daemons/bird/bgp.j2

```diff
@@ -70,15 +73,21 @@ protocol bgp bgp_{{ n.name }}_{{ af }} {
   rr cluster id {{ bgp.rr_cluster_id }};
 {%       endif %}
 {%     endif %}
-{%     if af in n.activate and n.activate[af] %}
-  {{ af }} {
+{%     for _af in [ 'ipv4','ipv6' ] if _af==af or (_af=='ipv4' and n.ipv4_rfc8950|default(False)) %}
+{%      if _af in n.activate and n.activate[_af] %}
```

Comment:
* I would prefer consistent indentation instead of squeezing another if block in between existing indentations.
* `n.activate[_af]|default(False)` might be cleaner than what you're using
* Finaly, it would be possible to combine everything into one **for ... if** block (and you can break it across multiple lines to make it easier to read).